### DISCUSSION TITLE
Using JitPack repository

### DISCUSSION THREAD

**Discussion by JJTech0130** [Hakky54/ayza#129] [Ideas]
Just posting this here in case anyone else finds it useful:
I wanted to use the latest commit of sslcontext-kickstarter (so that I could use `withUnsafeHostnameVerifier()`), but the maven central version only has tagged commits. The solution I found was to add JitPack: `maven("https://jitpack.io")` to the repositories. I could then add `implementation("com.github.Hakky54.sslcontext-kickstart:sslcontext-kickstart:master-SNAPSHOT")` to the dependencies.
(I'm using Kotlin DSL, so you might have to adapt it)

**Comment by JJTech0130** [comment]
Well, funnily enough, it turns out I don't even need `withUnsafeHostnameVerifier()`... but anyway 😄

**Comment by Hakky54** [comment]
Ah, thats a nice way to use the latest version without it being released to maven central. I didn't know this, thank you for sharing! I was planning to publish it but I was not able to do it. For the time being you could just use this with the tagged version:
```
SSLFactory.builder()
    .withHostnameVerifier(HostnameVerifierUtils.createUnsafe())
    .build()
```
or
```
SSLFactory.builder()
    .withHostnameVerifier((host, session) -> true)
    .build()
```
